core/embed/prodtest/main.c

```diff
@@ -703,6 +879,12 @@ int main(void) {
     } else if (startswith(line, "TOUCH ")) {
       test_touch(line + 6);
 
+    } else if (startswith(line, "TOUCH_CUSTOM ")) {
+      test_touch_custom(line + 13);
+
```

Comment:
All other two-word commands have no underscore between the words.

Reply:
see above